Second set of nut fact and acrolinx updates (#4767)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md

## Create groups

> Suppose you have groups in your user data that you want to include when you create your users in Okta. To do this, you must first create Okta Groups that are equivalent to the groups in your user data. You can create an Okta Group for the sample data (in this case, "All Employees") using an [Add Group](/docs/reference/api/groups/#add-group) request:

Reviewer: ```suggestion
Suppose you have groups in your user data that you want to include when you create your users in Okta. To do this, you must first create Okta Groups that are equivalent to the groups in your user data. You can create an Okta group for the sample data (in this case, "All Employees") using an [Add Group](/docs/reference/api/groups/#add-group) request:
```

You can obtain the new Group ID (`id`) from the response to use when you create Users in that Group later. You can also list all Groups in your org and obtain their IDs using a [List Groups](/docs/reference/api/groups/#list-groups) request.

> You can also create Groups in your Admin Console. For more information, see [About groups](https://help.okta.com/okta_help.htm?id=Directory_Groups) in the product documentation.

Reviewer: ```suggestion
You can also create groups in your Admin Console. For more information, see [About groups](https://help.okta.com/okta_help.htm?id=Directory_Groups) in the product documentation.
```

## Create users

> Once you’ve created all the necessary Okta Groups, you can create Users including their Group Memberships. As mentioned earlier, this example uses [Create User without Credentials](/docs/reference/api/users/#create-user-without-credentials) to create our sample user.

Reviewer: ```suggestion
Once you’ve created all the necessary Okta groups, you can create users, including their group memberships. As mentioned earlier, this example uses [Create User without Credentials](/docs/reference/api/users/#create-user-without-credentials) to create our sample user.
```

## User status and activation

> The user status in the response when you create a User is set to `STAGED`, which essentially means that the user has been created but not activated yet. You can activate users using the API or in your Admin Console. For more information on account states and activation, see:

Reviewer: ```suggestion
The user status in the response when you create a user is set to `STAGED`, which means that the user has been created but not activated yet. You can activate users using the API or in your Admin Console. For more information on account states and activation, see the following links:
```
